wdl-engine/src/http.rs

```diff
@@ -30,10 +32,15 @@ use tracing::debug;
 use tracing::info;
 use url::Url;
 
+use crate::config::Config;
+
+mod azure;
```

Comment:
Nit, `azure` isn't of like kind with `gs`/`s3`. I didn't like `azure_buckets`, but maybe just the protocol (`az`) would be slightly better? I definitely do not care too much either way, but it came across my mind.

Reply:
To make it more consistent and a little bit clearer from the file name, I renamed `gs` to `google`, so the outlier is now `s3`, but that's already a well-known numeronym.